src/QirRuntime/test/Standalone/InputTypes/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+# compile test ll files into a library
+
+set(INPUT_TYPES_FILES
+#  qir-test-arrays
+#  qir-test-functors
+#  qir-test-noqsharp
+#  qir-test-qsharp
+#  qir-test-partials
+#  qir-test-qubits-results
+  qir-input-types
+)
+
+foreach(file ${INPUT_TYPES_FILES})
```

Comment:
I presume, this sample will integrate with Q# projects fully and, thus, will always have a single input QIR file, correct? In that's the case, you could simplify this a bit.

Reply:
Simplified.